sdk/openai/openai-assistants/review/openai-assistants.api.md

```diff
@@ -0,0 +1,649 @@
+## API Report File for "@azure/openai-assistants"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { ClientOptions } from '@azure-rest/core-client';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationOptions } from '@azure-rest/core-client';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export interface Assistant {
+    createdAt: Date;
+    description: string | null;
+    fileIds: string[];
+    id: string;
+    instructions: string;
+    metadata: Record<string, string>;
+    model: string;
+    name: string | null;
+    object: "assistant";
+    tools: ToolDefinition[];
+}
+
+// @public
+export interface AssistantCreationOptions {
+    description?: string;
+    fileIds?: string[];
+    instructions?: string;
+    metadata?: Record<string, string>;
+    model: string;
+    name?: string;
+    tools?: ToolDefinition[];
+}
+
+// @public
+export interface AssistantDeletionStatus {
+    deleted: boolean;
+    object: "assistant.deleted";
+}
+
+// @public
+export interface AssistantFile {
+    assistantId: string;
+    createdAt: Date;
+    id: string;
+    object: "assistant.file";
+}
+
+// @public
+export interface AssistantFileDeletionStatus {
+    deleted: boolean;
+    object: "assistant.file.deleted";
+}
+
+// @public
+export interface AssistantImageFile {
+    file_id: string;
+}
+
+// @public
+export interface AssistantMessage {
+    assistantId?: string;
+    content: AssistantMessageContent[];
+    createdAt?: Date;
+    id?: string;
+    metadata?: Record<string, string>;
+    object: "thread.message";
+    role: AssistantRole;
+    runId?: string;
+    threadId?: string;
+}
+
+// @public
+export interface AssistantMessageContent {
+    // (undocumented)
+    file_ids?: string[];
```

Comment:
There seems to be a bunch of them

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-js/pull/27807/commits/04c1f3e123173e7618b7a9f104abd927ad9f2475)